x/profiles/keeper/keeper_chain_links_test.go

```diff
@@ -326,8 +369,15 @@ func (suite *KeeperTestSuite) TestKeeper_GetChainLink() {
 }
 
 func (suite *KeeperTestSuite) TestKeeper_DeleteChainLink() {
-	account := profilestesting.GetChainLinkAccount("cosmos", "cosmos")
-	secondAccount := profilestesting.GetChainLinkAccount("cosmos", "cosmos")
+	accounts := []profilestesting.ChainLinkAccount{
+		profilestesting.GetChainLinkAccount("cosmos", "cosmos"),
+		profilestesting.GetChainLinkAccount("cosmos", "cosmos"),
+		profilestesting.GetChainLinkAccount("cosmos", "cosmos"),
+	}
+	sort.SliceStable(accounts, func(i, j int) bool {
+		return accounts[i].Bech32Address().GetValue() < accounts[j].Bech32Address().GetValue()
+	})
```

Comment:
It requires at least three chain links to test update oldest default external address:
https://github.com/desmos-labs/desmos/blob/405f77bbe774ff3d2dd264f0f929959ffd1151cf/x/profiles/keeper/keeper_chain_links.go#L88-L91